dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/imports/validation/EnrollmentImportValidationTest.java

```diff
@@ -119,14 +118,6 @@ void testPreheatOwnershipForSubsequentEnrollment() throws IOException {
             });
   }
 
-  @Test
-  void testDisplayIncidentDateTrueButDateValueIsInvalid() {
-    // TODO(DHIS2-17768 tracker) what are we testing here? should we not call the importer?
-    assertThrows(
-        IOException.class,
-        () -> fromJson("tracker/validations/enrollments_error-displayIncident.json"));
```

Comment:
is this still something we should test, just not here?

Reply:
Yes, but this is not just tracker related and the error raised by this is not clear.
I think it can be discussed in the Thursady meeting.
i will add a point in the agenda